ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -1508,6 +1508,7 @@ MiFindInitializationCode(OUT PVOID *StartVa,
             {
                 /* Remember this */
                 InitFound = TRUE;
+                InitSection = Section;
```

Comment:
You're using this just for DPRINT right?

Reply:
Yes. I previously had another change that used it IIRC, but I removed it. I added a UNREFERENCED macro for release builds.